erc190/package.py

```diff
@@ -29,6 +29,23 @@ def __init__(self, lockfile):
 
         try:
             validate(package_data, schema_data)
-        except jsonValidationError:
+        except:
             raise ValidationError
         self.package_data = package_data
+
+    def __repr__(self):
+        """
+        This can be expanded to include the
+        hashed content as discussed once
+        Package.hash exists
+        """
+
+        name = self.package_data['package_name']
+        version = self.package_data['version']
+        return "<Package " + name + "==" + version + ">"
+
+    def name(self):
```

Comment:
Can you change these to have an `@property` decorator so that they can be accessed as `pkg.name` rather than `pkg.name()`

Reply:
I get the impression it's pretty basic, but I wasn't aware of the `@property` decorator before this. Thanks!